source/_integrations/bring.markdown

```diff
@@ -30,6 +31,13 @@ For authentication, the integration requires the `email` and `password` you used
 
 {% include integrations/config_flow.md %}
 
+## Sensors
+
+- **Urgent:** Shows the number of items tagged with the `Urgent` badge on the shopping list. Completed items are excluded.
+- **On occasion:** Displays the count of items marked with the `If convenient` badge.
+- **Discount only:** Indicates the number of items tagged with the `Offer` badge.
+- **Region & Language:** The sensor can be used for diagnostics. If everything is set correctly, it will display the selected Region for the shopping list. If it shows `Unknown`, the Region has not been set properly in the **Bring!** app.
```

Comment:
```suggestion
- **Region & Language:** The sensor can be used for diagnostics. If everything is set correctly, it will display the selected region for the shopping list. If it shows **Unknown**, the region has not been set properly in the **Bring!** app.
```